linera-core/src/client/mod.rs

```diff
@@ -2073,12 +2073,12 @@ where
     }
 
     /// Executes an operation.
-    #[instrument(level = "trace")]
```

Comment:
Note: keeping this would require the `operation` parameter to also have the `Debug` constraint. This would be possible, but I thought it was simpler to just remove it since this method just calls `execute_operations` anyway, which already has the tracing instrumentation as well.